src/components/PaginatedTable/index.tsx

```diff
@@ -5,26 +5,34 @@ import Table from '@/components/Table';
 import classNames from 'classnames';
 import ArrowLeft from '@/assets/images/icons/web/arrow-left.svg';
 import ArrowRight from '@/assets/images/icons/web/arrow-right.svg';
-import styles from '@/components/PaginatedTable/PaginatedTable.css';
 import { isEmpty } from 'lodash';
+import EmptyState from '../EmptyState';
+import styles from '@/components/PaginatedTable/PaginatedTable.css';
 
 export interface Props {
   currentPage: number;
+  emptyStateDescription?: string;
```

Comment:
please receive a component instead of configurations. It doesn't make sense to define in stone how we render empty states

Reply:
you can receive a function called renderEmptyState() that returns the component you want